projects/Mallard/src/helpers/download-todays-issue.ts

```diff
@@ -0,0 +1,15 @@
+import { prepFileSystem, clearOldIssues, downloadTodaysIssue } from './files'
```

Comment:
why is this file named `download-todays-issue.ts` when the `downloadTodaysIssue` function is in another file? 🤔

Reply:
There is some confusion with the naming, this is now changed.